### DISCUSSION TITLE
Keybinding to :set gutters '["X", "Y"]'

### DISCUSSION THREAD

**Discussion by vglfr** [helix-editor/helix#9768] [Q&A] [answered]
`"n" = ":set gutters []"` in config works fine.

`:set gutters '["X", "Y"]'` in command mode works fine.

`"N" = ":set gutters '[\"X\", \"Y\"]'"` in config is not working. Obviously an escape issue. Played with single quotes and multiline strings to no avail. The same error message "Bad arguments." Please help.

**Comment by the-mikedavis** [comment, marked as answer]
You can use a TOML literal string: `keys.normal.G = ''':set gutters ["diff","line-numbers"]'''`

**Reply by vglfr** [reply]
Thanks for the solution!

For those as confused as I am: dropping space between list members does the trick. `:set gutters ["diff","spacer"]` works fine in command mode while `:set gutters ["diff", "spacer"]` fails. Without spaces even single quoting works: `"N" = ':set gutters ["diff","spacer"]'`.
